rebar.config

```diff
@@ -8,10 +8,31 @@
     ]}
 ]}.
 
-%% Enables "rebar3 fmt" command
-{project_plugins, [erlfmt]}.
+{alias, [
+    {format, [
+        {do, "default as format fmt -w"}
+    ]},
+    {format_check, [
+        {do, "default as format fmt -c"}
+    ]},
+    {run_tests, [
+        {ct, "--sname=ct1"}
+    ]},
+    {cover_tests, [
+        {ct, "--sname=ct1 --cover"}
+    ]},
+    {all, [
+        format,
+        run_tests,
+        xref,
+        dialyzer
```

Comment:
You might want to add here hank or gradualizer or any other tool you might add later :)

Reply:
yes, right